CMakeLists.txt

```diff
@@ -174,6 +174,7 @@ endif()
 
 # we are making *a Python 3.6 Interpreter* a required dependency, so find it here
 # If LINK_WITH_PYTHON, also request the Development (libs) at the same time, to ensure consistent version between interpreter and Development
+# and ask for at least 3.8 (for the PyConfig stuff).
```

Comment:
I'd be almost inclined to just require 3.8 all the time, considering 3.6 died a long time ago, and 3.8 is about to go to end-of-life next year.  But this is clearly laying out the needs for our CPython uses.  :+1: